cdap-docs/reference-manual/source/release-notes.rst

```diff
@@ -26,19 +26,584 @@ Cask Data Application Platform Release Notes
+  from 'sink' to either 'batchsink' or 'realtimesink' to reflect that the plugins
+  implement different interfaces.
+
+- `CDAP-1554 <https://issues.cask.co/browse/CDAP-1554>`__ -
+  Moved constants for default and system namespaces from Common to Id.
+
+- `CDAP-3388 <https://issues.cask.co/browse/CDAP-3388>`__ -
+  Added interfaces to ``cdap-spi`` that abstract StreamEventRecordFormat (and dependent
+  interfaces) so users can extend the ``cdap-spi`` interfaces.
+
+- `CDAP-3583 <https://issues.cask.co/browse/CDAP-3583>`__ -
+  Added a RESTful API for retrieving the metadata associated with a particular run of a
+  CDAP program.
+  
+- `CDAP-3632 <https://issues.cask.co/browse/CDAP-3632>`__ -
+  Added a RESTful API for computing lineage of a CDAP dataset or stream.
+
+Deprecated and Removed Features
+-------------------------------
+
+- See the :ref:`CDAP 3.2.0 Javadocs <javadocs>` for a list of deprecated and removed APIs.
+
+- `CDAP-2667 <https://issues.cask.co/browse/CDAP-2667>`__ -
+  Removed application templates and adapters RESTful APIs, as these templates and adapters
+  have been replaced with applications that can be controlled with the Lifecycle HTTP
+  RESTful API.
+
+- `CDAP-3596 <https://issues.cask.co/browse/CDAP-3596>`__ -
+  Replaced the ETL ApplicationTemplates with the new ETL Applications.
+
 .. _known-issues-320:
 
 Known Issues
 ------------
 
+- `CDAP-3361 <https://issues.cask.co/browse/CDAP-3361>`__ -
+  CDAP-UI should use 'name' instead of 'id' fields when listing app programs.
+  
+- `CDAP-3396 <https://issues.cask.co/browse/CDAP-3396>`__ -
+  CDAP-UI scroll bar for logs has jitters and is unbounded at the top.
+  
+- `CDAP-3409 <https://issues.cask.co/browse/CDAP-3409>`__ -
+  java.lang.NoClassDefFoundError: co/cask/cdap/data/dataset/DatasetCreationSpec
+  
+- `CDAP-3492 <https://issues.cask.co/browse/CDAP-3492>`__ -
+  Executing 'select \*' from a dataset with many fields generates an error.
+  
+- `CDAP-3499 <https://issues.cask.co/browse/CDAP-3499>`__ -
+  TestFrameworkTestRun testGetServiceURL fails sporadically.
+  
+- `CDAP-3500 <https://issues.cask.co/browse/CDAP-3500>`__ -
+  TestFrameworkTestRun testWorkerStop fails sporadically on Bamboo builds.
+  
+- `CDAP-3534 <https://issues.cask.co/browse/CDAP-3534>`__ -
+  CDAP-UI metric shown in output graph in flowlets is not the correct one.
+  
+- `CDAP-3573 <https://issues.cask.co/browse/CDAP-3573>`__ -
+  Worker RuntimeException behavior in Standalone and Distributed are not the same.
+  
+- `CDAP-3578 <https://issues.cask.co/browse/CDAP-3578>`__ -
+  CDAP-UI Service explore dialog "view" icon is broken after download is clicked.
+  
+- `CDAP-3601 <https://issues.cask.co/browse/CDAP-3601>`__ -
+  Large number of Javadoc errors in modules
+  
+- `CDAP-3602 <https://issues.cask.co/browse/CDAP-3602>`__ -
+  The Documentation Search Engine does not find all pages with the search term.
```

Comment:
Well, then people should not be surprised if they can't find something...even if it actually is there.